c_src/eleveldb.cc

```diff
@@ -459,6 +465,45 @@ ERL_NIF_TERM parse_open_option(ErlNifEnv* env, ERL_NIF_TERM item, leveldb::Optio
                 opts.cache_object_warming = false;
         }
 
+        else if (option[0] == eleveldb::ATOM_EXPIRY_ENABLED)
+        {
+            if (option[1] == eleveldb::ATOM_TRUE)
+            {
+                if (NULL==opts.expiry_module.get())
```

Comment:
Should checking/creating the expiry_module member be moved outside the ATOM_TRUE conditional? That way, we set the expiry_enabled member, regardless of whether the expiry_module member was previously created. Then we don't have to worry about the order in which the various expiry options are declared or what their defaults are in the leveldb::ExpiryModuleOS class.

The same question/comment applies to the other expiry options below.